src/assets/styles/pages/courses.scss

```diff
@@ -0,0 +1,31 @@
+.course{
+    box-shadow: 0 0 3px rgba(0, 0, 0, 0.1);
+    cursor:pointer;
+    &__search{
+        display: flex;
+        justify-content: center;
+        margin: auto;
+        @media (max-width: 767px){
+            display: block;
+        }
+    }
+    &__page{
+        &--heading{
+            display: flex;
+            justify-content: center;
+            margin: 20px 10%;
+            @media (max-width: 767px){
+                flex-direction: column;
+                justify-content: flex-start;
+                margin: 40px auto;
+                text-align: center;
+            }
+        }
+    }
+}
+.courses__container{
+    margin: 50px 10%;
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(225px,1fr));
+    gap : 20px;
+}
```

Comment:
New line here

Reply:
Ok, will have to fix my ESlint i guess

Reply:
Just add a new line. ESLint does not work in SASS files.

Reply:
@eliemugenzi  ok. didn't properly check. let me fix it